### PR TITLE
Report PanZoomView

### PR DESCRIPTION
**Pull request by NicklasNielsen**

### CODE REVIEW

File: Report/docs/features/subfeatures/PanZoom.tex [hunk 1]
```diff
@@ -0,0 +1,7 @@
+\section{PanZoomView}
+
+PanZoomView is the first level under the View class, its responsibility is to handle panning, zooming of the view and touch on the screen.
```
Comment (JakobVork) [inline Report/docs/features/subfeatures/PanZoom.tex]: level? what? I get what you mean.. I think.. but I don't think a unknown reader would

File: Report/docs/features/subfeatures/PanZoom.tex [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+The onTouchEvent from the view class is overridden in the PanZoomView so that zooming can be supported through a ScaleGestureDetector. Every time the screen is touched onTouchEvent method is called to determine what kind of gesture is being made. Zoom happens through a listener that overrides the onScale function which changes the scaleFactor of the canvas and updates the map. When the map gets updated it is scaled after the scaleFactor.
```
Comment (JakobVork) [inline Report/docs/features/subfeatures/PanZoom.tex]: ScaleGestureDetector.. What's that?

Comment (JakobVork) [inline Report/docs/features/subfeatures/PanZoom.tex]: "..is touched THE onTouchEvent.."

File: Report/docs/features/subfeatures/PanZoom.tex [hunk 3]
```diff
@@ -0,0 +1,7 @@
+\section{PanZoomView}
+
+PanZoomView is the first level under the View class, its responsibility is to handle panning, zooming of the view and touch on the screen.
+
+The onTouchEvent from the view class is overridden in the PanZoomView so that zooming can be supported through a ScaleGestureDetector. Every time the screen is touched onTouchEvent method is called to determine what kind of gesture is being made. Zoom happens through a listener that overrides the onScale function which changes the scaleFactor of the canvas and updates the map. When the map gets updated it is scaled after the scaleFactor. 
+
+Panning is needed when the map is moved and therefor in the onTouchEvent function, if a move gesture is made and scaling is not in the process panning will be handled by setting the position of screen to the current coordinates minus when the gesture was a touch down and the updating the map.
```
Comment (Banders2) [inline Report/docs/features/subfeatures/PanZoom.tex]: therefor aint no laws this time.

Little bit technical on the last part, could maybe be left out:
 minus when the gesture was a touch down and the updating the map.

File: Report/docs/features/subfeatures/PanZoom.tex (hunk 2 above)
Comment (JakobVork) [inline Report/docs/features/subfeatures/PanZoom.tex]: I think it should be ".. is made" instead of ".. is being made". Sound a bit weird?

Comment (JakobVork) [inline Report/docs/features/subfeatures/PanZoom.tex]: ".. is scaled after the scaleFactor." should be changed to "is scaled based on the scaleFactor."

File: Report/docs/features/subfeatures/PanZoom.tex (hunk 3 above)
Comment (JakobVork) [inline Report/docs/features/subfeatures/PanZoom.tex]: therefore*

Comment (JakobVork) [inline Report/docs/features/subfeatures/PanZoom.tex]: I think you should use a dot instead of comma, since it actually is a new sentence.

Comment (JakobVork) [inline Report/docs/features/subfeatures/PanZoom.tex]: I did not understand this part though?

File: Report/docs/features/subfeatures/PanZoom.tex [hunk 4]
```diff
@@ -0,0 +1,7 @@
+\section{PanZoomView}
+
+PanZoomView extends the View class, its responsibility is to handle panning\footnote{https://en.wikipedia.org/wiki/Panning\_(camera)}, zooming of the view and touch on the screen.
```
Comment (RasmusHemmingsen) [inline Report/docs/features/subfeatures/PanZoom.tex]: ikke its, det lyder forkert

**Pull request merged**